bin/telex2reddit

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python3
+
+import telex2reddit
+
+if __name__ == '__main__':
+    telex2reddit.init()
```

Comment:
Kicsit modularizáltam, a cél a több száz soros függvények eliminálása (kisebbekbe refaktorálása), illetve majd a globálisokat is át kell alakítani.